package.json

```diff
@@ -102,14 +102,14 @@
     "copy-text-to-clipboard": "1.0.4",
     "cozy-bar": "7.7.6",
     "cozy-ci": "0.4.1",
-    "cozy-client": "7.1.0",
+    "cozy-client": "13.2.0",
     "cozy-client-js": "0.16.4",
     "cozy-device-helper": "1.8.0",
     "cozy-doctypes": "1.67.0",
     "cozy-flags": "1.10.0",
     "cozy-realtime": "2.0.8",
     "cozy-scripts": "1.13.2",
-    "cozy-ui": "25.4.1",
+    "cozy-ui": "33.3.1",
```

Comment:
Are you up to date with master? It seems we're already on 33.3.1 https://github.com/cozy/cozy-drive/blob/master/package.json#L118 ?

Reply:
Thanks, I forgot to pull before developing.